tests/users/test_data.py

```diff
@@ -174,3 +174,15 @@ async def test_find_or_create_b2c_user(
         assert int(user.handle.split("-")[-1])
 
     assert user == snapshot(matcher=path_type({"handle": (str,)}))
+
+
+async def test_check_is_empty(mongo, users_data, fake2, authorization_client, pg):
+    first_user_exists = not await users_data.check_is_empty()
+
+    assert first_user_exists is False
```

Comment:
Variable is unnecessary. Using `not` and `False` just adds to cognitive load. Be direct.

Same goes for the other place you do this.
```suggestion
    assert await users_data.check_is_empty() is True
```